src/modules/utils/player/Player.h

```diff
@@ -50,8 +50,8 @@ class Player : public Module {
         StreamOutput* suspend_stream;
 
         FILE* current_file_handler;
-        unsigned long file_size, played_cnt;
-        unsigned long elapsed_secs;
+        long file_size, played_cnt;
+        long elapsed_secs;
```

Comment:
why is elapsed seconds long now? i mace them unsigned for a reason. i understand why file size is long bit i disagree with that too as the error should be detected at source.